src/test/python/CMakeLists.txt

```diff
@@ -14,50 +14,60 @@ set(included_namespaces
     Windows.Foundation
     Windows.Graphics.DirectX
 
+
     # uncomment out the following to run complete_winml_tutorial.py
-    Windows.AI.MachineLearning
-    Windows.Graphics.Imaging
-    Windows.Media
-    Windows.Storage
+    # Windows.AI.MachineLearning
```

Comment:
Why not just remove these lines? Just curious.

Reply:
so I can easily uncomment them to run the complete_winml_tutorial test